### PR TITLE
Add simple 3-big-buttons editor for gray places

### PR DESCRIPTION
**Pull request by opyh**

### CODE REVIEW

File: src/components/NodeToolbar/AccessibilityEditor/SimpleWheelchairAccessibilityEditor.js [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+    &:hover {
+      background-color: ${colors.negativeBackgroundColorTransparent};
+    }
+    &:active {
+      background-color: ${colors.negativeColor};
+    }
+  }
+`;
+
+export default function SimpleWheelchairAccessibilityEditor(props: Props) {
```
Comment (mutaphysis) [inline src/components/NodeToolbar/AccessibilityEditor/SimpleWheelchairAccessibilityEditor.js]: Maybe: Inline...Editor

File: src/components/NodeToolbar/AccessibilityEditor/WheelchairStatusEditor.js [hunk 2]
```diff
@@ -38,12 +39,13 @@ class WheelchairStatusEditor extends React.Component<Props, State> {
   props: Props;
 
   state = {
-    wheelchairAccessibility: 'unknown',
+    wheelchairAccessibility: null,
```
Comment (mutaphysis) [inline src/components/NodeToolbar/AccessibilityEditor/WheelchairStatusEditor.js]: -1: state type does not allow null value, please check for flow errors.

File: src/components/NodeToolbar/AccessibilityEditor/WheelchairStatusEditor.js [hunk 3]
```diff
@@ -38,12 +39,13 @@ class WheelchairStatusEditor extends React.Component<Props, State> {
...
     categoryId: 'other'
   };
 
   constructor(props) {
     super(props);
+    debugger
```
Comment (mutaphysis) [inline src/components/NodeToolbar/AccessibilityEditor/WheelchairStatusEditor.js]: -2: Remove

File: src/components/NodeToolbar/AccessibilityEditor/WheelchairStatusEditor.js [hunk 4]
```diff
@@ -25,7 +25,8 @@ type Props = {
   className: string,
   onSave: ?((YesNoLimitedUnknown) => void),
   onClose: (() => void),
-  inline: ?boolean 
+  presetStatus: YesNoLimitedUnknown,
```
Comment (mutaphysis) [inline src/components/NodeToolbar/AccessibilityEditor/WheelchairStatusEditor.js]: Maybe: Allow undefined or null here as well

File: src/components/NodeToolbar/NodeToolbar.js [hunk 5]
```diff
@@ -243,6 +248,28 @@ class NodeToolbar extends React.Component<Props> {
...
+    if (!wheelmapFeature || !wheelmapFeature.properties) {
+      return;
+    }
+    if (wheelmapFeature.properties.wheelchair !== 'unknown') {
+      return;
+    }
+
+    return <section>
+      <h4 id="wheelchair-accessibility-header">
+        {t`How wheelchair accessible is this place?`}
```
Comment (mutaphysis) [inline src/components/NodeToolbar/NodeToolbar.js]: Nitpick: Weird, german sounding phrasing

**Comment by opyh** [comment]
Done!

**Pull request merged**